C++/Dynamic-Programming/README.md

```diff
@@ -1,3 +1,5 @@
+# Dynamic Programming
```

Comment:
Remove this File

Reply:
What about C folder ?

Reply:
I am also removing C folder Dp readme.md